lib/matplotlib/transforms.py

```diff
@@ -682,6 +682,41 @@ def intersection(bbox1, bbox2):
 class Bbox(BboxBase):
     """
     A mutable bounding box.
+
+    Examples
+    --------
+    >>> # the default constructor takes the boundary "points" [[xmin, ymin],
```

Comment:
Do you need to mark as code _and_ a comment? Can you remove the `>>> #` for direct text?

Reply:
This is what happens when you open a new PR at 4am, fixed.